pages/slider/permutations.page.tsx

```diff
@@ -2,44 +2,146 @@
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
 
+import { Box } from '~components';
 import Slider, { SliderProps } from '~components/slider';
 
 import createPermutations from '../utils/permutations';
 import PermutationsView from '../utils/permutations-view';
 import ScreenshotArea from '../utils/screenshot-area';
 
-const permutations = createPermutations<SliderProps>([
+const permutations = createPermutations<SliderProps & { label: string }>([
   {
+    label: ['Value'],
     value: [-10, 0, 10],
     min: [-10],
     max: [10],
-    hideFillLine: [false, true],
-    disabled: [false, true],
-    tickMarks: [false, true],
-    invalid: [false, true],
-    step: [0.5, 1, 5],
```

Comment:
I think it would be nice to keep covering 0.5 as a value for `step `

Reply:
updated

Reply:
Yes, just did it